ext/dom/xpath.c

```diff
@@ -154,11 +154,11 @@ static void dom_xpath_ext_function_php(xmlXPathParserContextPtr ctxt, int nargs,
 	fci.retval = &retval;
 
 	if (!zend_make_callable(&fci.function_name, &callable)) {
-		php_error_docref(NULL, E_WARNING, "Unable to call handler %s()", ZSTR_VAL(callable));
+		zend_throw_error(NULL, "Unable to call handler %s()", ZSTR_VAL(callable));
```

Comment:
I expect this will leak, at least function_name is allocated above

Reply:
This indeed was leaking, I tried to amend the domxpath.phpt test,  but it only covers one case. Maybe @beberlei knows how to trigger the other cases?

Reply:
looks like:

```php
$xpath->registerPhpFunctions(['non_existant']);
```